### PR TITLE
Release/7.3.0

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version=7.2.0
+version=7.3.0-rc2
```
Comment (jeremyjams) [inline gradle.properties:1]:
```suggestion
version=7.3.0
```

File: README.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ To run properly, the iExec Blockchain Adapter API requires:
 | IEXEC_BLOCKCHAIN_ADAPTER_API_WALLET_PATH | Path to the wallet of the server. | String | `src/main/resources/wallet.json` |
 | IEXEC_BLOCKCHAIN_ADAPTER_API_WALLET_PASSWORD | Password to unlock the wallet of the server. | String | `whatever` |
 
+## Health checks
+
+A `/actuator/health` endpoint is enabled by default and can be accessed on the **IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
```
Comment (jeremyjams) [inline README.md:41]:
```suggestion
An `/actuator/health` endpoint is enabled by default and can be accessed on the **IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
```

**Review by jeremyjams — APPROVED** [review]
Thank you :+1:

File: README.md (hunk 2 above)
Comment (mcornaton) [reply README.md:41]: I would use `A`, as JB does. We would pronounce it `a slash actuator [...]`, not `an slash actuator [...]`.

**Review by mcornaton — APPROVED** [review]
Thanks!

File: README.md (hunk 2 above)
Comment (jeremyjams) [reply README.md:41]: Up to you guys :) (Slash is formatted at read time)

Comment (jbern0rd) [reply README.md:41]: What about this ?
```suggestion
An health endpoint (`/actuator/health`) is enabled by default and can be accessed on the **IEXEC_BLOCKCHAIN_ADAPTER_API_PORT**.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[7.3.0]](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/releases/tag/v7.3.0) 2023-01-18
+
+* Add health endpoint to allow health checks mechanism enablement.
```
[the remark this answers is not in the document]
Comment (jbern0rd) [reply CHANGELOG.md:7]: Maybe simply
```suggestion
* Add health endpoint to allow health checks.
```

File: gradle.properties (hunk 1 above)
Comment (jbern0rd) [reply gradle.properties]: Done in 8eb1575

File: README.md (hunk 2 above)
Comment (jbern0rd) [reply README.md]: Done in 3787b79

File: CHANGELOG.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jbern0rd) [reply CHANGELOG.md]: Maxence's suggestion used in 3787b79

**Review by mcornaton — APPROVED** [review]

**Review by jeremyjams — APPROVED** [review]
:+1:

**Pull request merged**